CRM/Financial/BAO/ExportFormat/IIF.php

```diff
@@ -351,7 +351,7 @@ public static function format($s, $type = 'string') {
     $s1 = str_replace(self::$SEPARATOR, '\t', $s);
     switch ($type) {
       case 'date':
-        $sout = date('Y/m/d', strtotime($s1));
+        $sout = CRM_Utils_Date::customFormat(date('Ymd', strtotime($s1)));
```

Comment:
Dilip, you don't seem to have paid much attention to the description of the problem: the suggested change will only remove '/' from the date, not change the order of the parts of the date as requested. 

More generally, the problem is due to different date formats being used in different countries, and by different users in fact, since these are configurable settings in most products like CiviCRM and QuickBooks. The complaint is that the date format output is not the same as what he has configured in CiviCRM. He's likely referencing the setting at Administer > Localization > Date Formats, Complete Date fields. 

Unfortunately, those settings don't include a convenient one for this purpose, since by default they include text names of months, etc.

The date settings at Administer > Custom Data and Screens > Date Preferences don't deal with localization, but could perhaps help this along. However, as this is mostly a problem with localization, rather than adding an entry, let's add it to Administer > Localization > Date Formats. Use 'Financial Batch Date' as the field label, and store in a new setting. Ask Pradeep for more details about a) creating the setting for a tarball, b) ensuring it is available on upgrade (this is now transparent, IIRC), c) retrieving and using here. Could you make its default value the US standard of MM/DD/YYYY?